add-promotion.php

```diff
@@ -0,0 +1,153 @@
+                                                <div class="col-6">
+                                                    <input type="radio" class="w-100" name="status" value="disable"> Disable
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                                <div class="col-12 mt-3">
+                                    <div class="row">
+                                        <div class="col-12 col-md-4 col-lg-4">
+                                            <label class="descriptions">Image</label>
+                                        </div>
+                                        <div class="col-12 col-md-8 col-lg-8">
+                                            <div class="row">
+                                                <div class="col-12">
+                                                    <div class="row">
+                                                        <div class="col-12">
+                                                            <input class="form-control w-100" type="file" id="formFile" accept=".png, .jpg, .jpeg" onclick="imagePreview();">
+                                                        </div>
+                                                        <div class="col-12 mt-2">
+                                                            <center><img alt="" id="des-image" style="width: 300px; height: 400px; object-fit: cover;" class="img-fluid rounded-2" src="resources/images/default -image.svg"></center>
+                                                        </div>
+                                                    </div>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                                <div class="col-12 mt-3">
+                                    <div class="row">
+                                        <div class="col-12 col-md-6 col-lg-6 mt-2">
+                                            <button class="btn sbt-button">Clear</button>
+                                        </div>
+                                        <div class="col-12 col-md-6 col-lg-6 mt-2">
+                                            <button class="btn sbt-button" onclick="newPromotion();">ADD</button>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                        <?php include 'back-footer.php'; ?>
+                    </div>
+                </div>
+            </div>
+        </div>
+    <?php
+    }
```

Comment:
Include the `else` code fragment and redirect to the proper page when the user session object is missing.